packages/messages/src/browser/notifications.ts

```diff
@@ -72,8 +74,16 @@ export class Notifications {
                 button.addEventListener('click', () => {
                     action.fn(handler);
                     close();
+                    clearTimeout(this.closeTimer);
```

Comment:
@lmcbout here you're referencing the `closeTimer` property of the `Notifications` instance. this is not, what you want to say it should do, right? you want to clear the timeout of the single message, but this way each click handler will clear the timeout of the latest shown message. I think I already commented on this, but GH comments were rooted out.

Reply:
Craete a clearTimeout() and setTimeout, so I can use the window.setTimeout and window.cearTimeout() method

Reply:
No need to create the set and clear timeout method, using wndow.set instead